u_tls_extensions.go

```diff
@@ -503,12 +503,10 @@ func (e *StatusRequestV2Extension) Write(b []byte) (int, error) {
 	extData := cryptobyte.String(b)
 	// RFC 4366, Section 3.6
 	var statusType uint8
-	var ignored cryptobyte.String
-	if !extData.ReadUint16LengthPrefixed(&ignored) ||
+	if !extData.ReadUint16LengthPrefixed(&extData) ||
 		!extData.ReadUint8(&statusType) ||
-		!extData.ReadUint16LengthPrefixed(&ignored) ||
-		!extData.ReadUint16LengthPrefixed(&ignored) ||
-		!extData.ReadUint16LengthPrefixed(&ignored) {
+		!extData.ReadUint16LengthPrefixed(&extData) ||
+		!extData.ReadUint16LengthPrefixed(&extData) {
```

Comment:
These two lines are not needed actually since we are not parsing. And the overriding makes it a bit confusing. We should either read them into `&ignored` or just remove these lines.

Reply:
I got rid of `ignore` fwiw since it wasnt used anywhere and just let cryptobyte overwrite `extData` on itself

Reply:
That said, what would you prefer? want me to just remove those 2 checks?

Reply:
It is just a notation that we are intentionally discarding those bytes. uTLS is not optimized for resource usage nor intended for performance-max anyways... 

But if you do care, I would suggest adding a comment that we are not using any of the other bytes, so that other contributors could easily understand what we are doing rather than seeing this as a potential typo/bug. (Especially the last `ReadUint16LengthPrefixed` which caused data loss)